.github/workflows/coverity.yml

```diff
@@ -0,0 +1,39 @@
+# Creates and uploads a Coverity build on a schedule
```

Comment:
We probably don't need this file in the release flows.

Reply:
We could add condition to run on unstable only.

Reply:
This file will ever get run anyways, as it's not on the default branch and it's executed in a schedule.